crates/cli/src/commands/init.rs

```diff
@@ -1,12 +1,137 @@
-use axiom_sdk::{AxiomConfig, DEFAULT_CONFIG_ID, STAGING_DEFAULT_CONFIG_ID};
 use clap::Parser;
 use eyre::Result;
+use std::fs;
+use std::path::Path;
+use std::process::Command;
 
-const STAGING_API_URL: &str = "https://api.staging.app.axiom.xyz/v1";
-const PROD_API_URL: &str = "https://api.axiom.xyz/v1";
+const MAIN_RS_PREPEND: &str = r#"#[allow(unused_imports)]
+use {
+    openvm_k256::Secp256k1Point,
+    openvm_p256::P256Point,
+    openvm_pairing::{
+        bls12_381::{Bls12_381Fp2, Bls12_381G1Affine},
+        bn254::{Bn254Fp2, Bn254G1Affine},
+    },
+};
+
+openvm::init!();
+
+"#;
+
+fn generate_additional_deps(tag: Option<&str>) -> String {
+    let tag_part = tag.map_or(String::new(), |tag| format!(r#", tag = "{}""#, tag));
+
+    format!(
+        r#"
+openvm-algebra-guest = {{ git = "https://github.com/openvm-org/openvm.git"{}, default-features = false }}
+openvm-ecc-guest = {{ git = "https://github.com/openvm-org/openvm.git"{}, default-features = false }}
+openvm-pairing = {{ git = "https://github.com/openvm-org/openvm.git"{}, features = [
+    "bn254",
+    "bls12_381",
+] }}
+openvm-k256 = {{ git = "https://github.com/openvm-org/openvm.git"{}, package = "k256" }}
+openvm-p256 = {{ git = "https://github.com/openvm-org/openvm.git"{}, package = "p256" }}
+"#,
+        tag_part, tag_part, tag_part, tag_part, tag_part
+    )
+}
+
+const OPENVM_TOML_TEMPLATE: &str = r#"openvm_version = "v1.2"
```

Comment:
reminder to update it to "v1.3" when we will deploy it (I will deploy today or tomorrow)